src/targets/objc/native.js

```diff
@@ -1,57 +1,76 @@
 'use strict';
 
 var util = require('util');
+var objcHelpers = require('./helpers');
 
 module.exports = function (source, options) {
   var opts = util._extend({
-    timeout: '10'
+    timeout: '10',
+    indent: true
   }, options);
 
   var code = [];
 
-  // Dependencies
   code.push('#import <Foundation/Foundation.h>');
+
+  if (Object.keys(source.allHeaders).length) {
+    code.push(null);
+    code.push(objcHelpers.nsDictionaryBuilder('headers', source.allHeaders, opts.indent));
+  }
+
+  var parameters;
```

Comment:
It's an object. It's declared as undefined. If there's something, it'll be an object. Seems better than declaring a boolean and later putting an object into it.